packages/trip-form/src/MetroModeSelector/index.tsx

```diff
@@ -345,6 +346,12 @@ function ModeButton({
             {...getFloatingProps()}
             // Matches ID on Header element in SubSettingsPane
             aria-labelledby={`metro-mode-selector-${modeButton.key}-button-label`}
+            // This is a workaround for a bug in Firefox.
+            // https://github.com/floating-ui/floating-ui/issues/2299
```

Comment:
Can we add a comment to keep track of the Firefox bug itself? https://bugzilla.mozilla.org/show_bug.cgi?id=1829500